plugins/gradle/java/testSources/compiler/GradleCompilingTestCase.java

```diff
@@ -10,13 +10,15 @@
 
 import java.io.File;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
 /**
  * @author Vladislav.Soroka
  */
 public abstract class GradleCompilingTestCase extends GradleImportingTestCase {
 
   protected void assertCopied(String path) {
-    assertTrue(file(path).exists());
+    assertThat(file(path)).exists();
```

Comment:
Better assertion messages

Reply:
I actually learned about assertj from intellij repo, so this is coming around full circle 👍